src/routes/+page.svelte

```diff
@@ -1,2 +1,19 @@
 <h1>Welcome to SvelteKit</h1>
 <p>Visit <a href="https://kit.svelte.dev">kit.svelte.dev</a> to read the documentation</p>
+<button class="btn">asdfasd</button>
+
+<div class="card w-96 bg-base-100 shadow-xl">
+    <figure>
+        <img
+            src="https://daisyui.com/images/stock/photo-1606107557195-0e29a4b5b4aa.jpg"
+            alt="Shoes"
+        />
+    </figure>
+    <div class="card-body">
+        <h2 class="card-title">Shoes!</h2>
+        <p>If a dog chews shoes whose shoes does he choose?</p>
+        <div class="card-actions justify-end">
+            <button class="btn btn-primary">Buy Now</button>
+        </div>
+    </div>
+</div>
```

Comment:
Could you revert back to the original page? It seems the shoe card that pops up was an example from when you were testing it out.

Reply:
Re-pushed after removing this code